src/deepEqual.js

```diff
@@ -24,7 +24,63 @@
  */
 
 function deepEqual(a, b) {
-  // write code here
+  if ((a === null && b !== null) || (a !== null && b === null)) {
+    return false;
+  }
+  if (typeof (a) === 'object' && typeof (b) === 'object') {
+    let countA = 0;
+    let countB = 0;
+    let strA = '';
+    let strB = '';
+    let numberA = 0;
+    let numberB = 0;
+    for (let key in a) {
+      countA++;
+      if (a[key] === 'object') {
+        deepEqual(a[key]);
```

Comment:
deepEqual is called with one argument (it requires 2 arguments). And result of calling deepEqual is ignored. this line is doing nothing.